service/lib/agama/storage/config_conversions/from_json_conversions/volume_group.rb

```diff
@@ -70,6 +61,40 @@ def convert_extent_size
             Y2Storage::DiskSize.new(value)
           end
 
+          # @return [Array<String>, nil]
+          def convert_physical_volumes_devices
+            generate_json = physical_volume_generate_json
+            return unless generate_json
+
+            generate_json.dig(:generate, :targetDevices)
```

Comment:
Didn't we agree to also implement a short version like this for those cases in which we don't need encryption settings?

```json
"volumeGroups": [
  {
    "name": "system",
    "physicalVolumes": [
      { "generate": ["vda"] }
    ],
  }
]
```

Kind of similar to the short versions of the `generate` for partitions and LVs.